src/checkers/zxcvbn.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import zxcvbn from "zxcvbn";
+
+const formatFeedback = feedback => {
+  if (!feedback.warning && !feedback.suggestions) {
+    return <p></p>;
+  }
+  return (
+    <>
+      <p>{feedback.warning}</p>
+      {feedback.suggestions && (
+        <ol>
+          {feedback.suggestions.map(suggestion => (
+            <li key={suggestion}>{suggestion}</li>
+          ))}
+        </ol>
+      )}
+    </>
+  );
+};
+const determineStrength = password => {
+  const { feedback, score } = zxcvbn(password);
+  return { strength: (score + 1) * 2, description: formatFeedback(feedback) };
```

Comment:
`zxcvbn` returns a score from 0 to 4, so this is just a simple way to convert it to a 10 point scale.